packages/core/examples/inputGroupExample.tsx

```diff
@@ -86,14 +82,16 @@ export class InputGroupExample extends BaseExample<IInputGroupExampleState> {
             <div>
                 <div>
                     <InputGroup
+                        className={className}
                         disabled={disabled}
                         leftIconName="filter"
                         onChange={this.handleFilterChange}
                         placeholder="Filter histogram..."
-                        rightElement={clearButton}
+                        rightElement={filterValue ? <Spinner className={Classes.SMALL} /> : undefined}
```

Comment:
nit: Again I find myself wishing for a `maybeRender...` helper when I read through this, even though it would be used only in this one place. Easier to read quickly.